src/app/clusters/laundry-washer-controls-server/laundry-washer-controls-server.h

```diff
@@ -0,0 +1,106 @@
+/**
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "laundry-washer-controls-delegate.h"
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+#include <app/util/af.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace LaundryWasherControls {
+
+/**
+ * @brief LaundryWasherControls Server Plugin class
+ */
+class LaundryWasherControlsServer : public AttributeAccessInterface
+{
+public:
+    LaundryWasherControlsServer() : AttributeAccessInterface(Optional<EndpointId>::Missing(), LaundryWasherControls::Id) {}
+    static LaundryWasherControlsServer & Instance();
+
+    using SpinSpeedList       = DataModel::List<const chip::CharSpan>;
+    using SupportedRinsesList = DataModel::List<const NumberOfRinsesEnum>;
+
+    static constexpr uint8_t kMaxSpinSpeedLength = 64;
```

Comment:
I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/7340 on the spec about the fact that this max size is not defined there.

Reply:
I just realized this that I will report this